<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>테스트 데이터 업로드</string>
    </property>
    <widget class="QComboBox" name="comboBox_jongmok2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>종목명</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>일자</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_searchDate">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>8</month>
       <day>2</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2022</year>
       <month>8</month>
       <day>2</day>
      </date>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2010</year>
       <month>9</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="maximumDate">
      <date>
       <year>2023</year>
       <month>12</month>
       <day>31</day>
      </date>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::MonthSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
     <property name="currentSectionIndex">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_testStart">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>데이터로드</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit_searchDate_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>8</month>
       <day>3</day>
      </datetime>
     </property>
     <property name="date">
      <date>
       <year>2022</year>
       <month>8</month>
       <day>3</day>
      </date>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::MonthSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd hh:mm:ss</string>
     </property>
     <property name="currentSectionIndex">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>~</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Californian FB</family>
      </font>
     </property>
     <property name="text">
      <string>※ 코인 시간으로 조회하기 위해서는 현재시간에서 9시간 더해서 조회</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_all">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>수동</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>자동</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="push_start">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>백테스트시작</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
